pages/_app.tsx

```diff
@@ -83,11 +84,19 @@ function AppUtilities({ children }: { children: ReactNode }) {
 }
 function Routing({ Component, pageProps }: AppProps) {
   const [store, setStore] = useState<Store | null>(null);
+  const { update: updateLogProtocol } = useFeature("logProtocol");
   const { getFeatureFlag } = useLaunchDarkly();
 
   useEffect(() => {
+    // Check what the initial value of logProtocol should be
+    if (window) {
```

Comment:
I think this would need to be `typeof window !== "undefined"` to avoid causing a runtime error. (I assume that's what's breaking the tests?)